Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml

```
@@ -191,7 +191,7 @@
   parent: BaseSignDirectional
   id: SignDirectionalSalvage
   name: salvage sign
-  description: A direction sign, pointing out which way the Salvage department is.
+  description: A direction sign, pointing out which way Salvage is.
```

A suggestion for avoiding the lone word on its own line.

```suggestion
  description: A direction sign, pointing out which to Salvage.
```